tensorflow_probability/python/optimizer/lbfgs.py

```diff
@@ -79,6 +79,7 @@
 
 def minimize(value_and_gradients_function,
              initial_position,
+             initial_state=None,
```

Comment:
let's call this `previous_optimizer_results`, which mirrors similar functionality and naming style in the MCMC package (see `previous_kernel_results` argument to `sample_chain`).

Reply:
Done, thanks!